CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 11.3.0
+### 🚀 New Features
+- `number` (https://github.com/primer/octicons/pull/541)
```

Comment:
Oh, I forgot to mention that this will also be in this release:

```suggestion
- `number` (https://github.com/primer/octicons/pull/541)

### 🐛 Fixes
- Fix rendering of arrows in Safari (@aaronshekey https://github.com/primer/octicons/pull/527)
```